inim.nimble

```diff
@@ -18,4 +18,5 @@ task test, "Run all tests":
   exec "mkdir -p bin"
   exec "nim c -d:NoColor -d:prompt_no_history --out:bin/inim src/inim.nim"
   exec "nim c -r -d:prompt_no_history tests/test.nim"
+  exec "nim c -r -d:INCLUDED -d:prompt_no_history tests/test_commands.nim"
```

Comment:
what does `-d:INCLUDED` do?

Reply:
Haha, I'll rename that, it's unnecessary